src/services/Services.jsx

```diff
@@ -1,23 +1,36 @@
+const matchLabsHost = "https://match-labs-api.herokuapp.com/api/no_auth"
+
 const getData = async (type) => {
-    // const urlLabs = "https://match-labs-api.herokuapp.com/api/no_auth/labs";
-    // const urlCandidates = "https://match-labs-api.herokuapp.com/api/no_auth/candidates";
-    const url = type === "labs" ? "https://match-labs-api.herokuapp.com/api/no_auth/labs" 
-                                : type === "candidates" ? "https://match-labs-api.herokuapp.com/api/no_auth/candidates"
-                                                        : "";
-    const data = await fetch(url);
-    const json = await data.json();
-    return json;
-}
+    const url = `${matchLabsHost}/${type}`
+    const rawData = await fetch(url);
+    const json = await rawData.json();
 
-const getDataById = async (type, id) => {
-    const url = type === "labs" ? `https://match-labs-api.herokuapp.com/api/no_auth/labs/${id}`
-                                : type === "candidates" ? `https://match-labs-api.herokuapp.com/api/no_auth/candidates/${id}`
-                                                        : "";
-    const data = await fetch(url);
-    const json = await data.json();
-    return json;
+    const data = json.map(item => {
+        const d = {}
```

Comment:
To prevent `d` duplication, you can construct the object directly!
```js
json.map(item => ({
  id: item.id,
  name: type === 'labs' ? item.name : `${item.first_name} ${item.last_name}`
  // ...
}))
```